backend/src/Installer.ts

```diff
@@ -152,71 +186,129 @@ export default class Installer {
     return name + platform
   }
 
-  binaryPath(admin?: boolean) {
+  get downloadFileNameMuxer() {
+    let platform = ''
+    if (environment.isWindows32) platform = 'muxer.x86-win.exe'
+    else if (environment.isWindows) platform = 'muxer.x86_64-64.exe'
+    else if (environment.isMac) platform = 'muxer.x86_64-osx'
+    return platform
+  }
+
+  get downloadFileNameDemuxer() {
+    let platform = ''
+    if (environment.isWindows32) platform = 'demuxer.x86-win.exe'
+    else if (environment.isWindows) platform = 'demuxer.x86_64-64.exe'
+    else if (environment.isMac) platform = 'demuxer.x86_64-osx'
+    return platform
+  }
+
+  get downloadFileNameConnectd() {
+    let platform = ''
+    if (environment.isWindows32) platform = 'connectd.x86-win.exe'
+    else if (environment.isWindows) platform = 'connectd.x86_64-64.exe'
+    else if (environment.isMac) platform = 'connectd.x86_64-osx'
+    return platform
+  }
+
+  binaryPathCLI(admin?: boolean) {
```

Comment:
Yea we can do this without repeating ourselves. Wanna keep the code ad DRY as possible.  🙂